cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -77,6 +77,12 @@ spec:
   # enable the crash collector for ceph daemon crash collection
   crashCollector:
     disable: false
+  cleanupPolicy:
+    # cleanupPolicy should only be added to the cluster when the cluster is about to be deleted.
+    # After any field of the cleanup policy is set, Rook will stop configuring the cluster as if the cluster is about
+    # to be destroyed in order to prevent these settings from being deployed unintentionally.
+    deleteDataDirOnHosts: ""
+    wipeDevices: ""
```

Comment:
Let's save wipeDevices for a later PR when we're ready to implement that. Maybe it will come soon, it just shouldn't be in the cluster CR until it's implemented.